analyzer/src/test/kotlin/PackageManagerTest.kt

```diff
@@ -29,7 +29,7 @@ import io.kotlintest.specs.WordSpec
 import java.io.File
 
 class PackageManagerTest : WordSpec({
-    val projectDir = File("analyzer/src/funTest/assets/projects/synthetic/all-managers")
+    val projectDir = File("../analyzer/src/funTest/assets/projects/synthetic/all-managers")
```

Comment:
Just omit "../analyzer/" from the start of the string.

Reply:
Done.